include/librealsense2/rs.hpp

```diff
@@ -36,6 +36,13 @@ namespace rs2
         rs2_reset_logger(&e);
         error::handle(e);
     }
+
+    inline void enable_rolling_files(std::size_t max_size )
```

Comment:
Not clear what the unit of measure is for max_size -- is it bytes? KB?

Reply:
done